docs/security/user-certs.md

```diff
@@ -140,10 +140,27 @@ export CIGETCERTOPTS="-i 'Fermi National Accelerator Laboratory'"
 Your VO may also provide specific instructions for how to best use this tool.
 Contact your VO support center for details.
 
-Finally, cigetcert has advanced options for things like selecting an auth method, loading a configuration from a server, or storing the cert on a MyProxy server.
+Finally, `cigetcert` has advanced options for things like selecting an auth method, loading a configuration from a server, or storing the cert on a MyProxy server.
 See the [manual page for cigetcert](http://htmlpreview.github.io/?https://github.com/fermitools/cigetcert/blob/master/cigetcert.html) for more information.
 
 
+### Using cigetcert with XSEDE credentials
+
+`cigetcert` also works with XSEDE as the service provider.
+To use XSEDE credentials, you will first need an account at <https://portal.xsede.org>.
+In addition, you _need_ to set up two-factor authentication with XSEDE; see their [MFA documentation](https://portal.xsede.org/mfa) for details.
+Push notifications using the Duo Mobile app are strongly recommended; the login prompt times out too quickly for a phone call to work.
```

Comment:
Let's not even mention the phone call option and say that Duo is a requirement for the cigetcert + xsede method